charts/xrd-vrouter/values.yaml

```diff
@@ -239,12 +239,11 @@ mgmtInterfaces: []
 # More information about the CPU behavior of XRd vRouter can be found
 # here: @@@
 cpu: {}
-  # Override the cpuset to use for XRd.
-  # This must be a subset of the CPUs available for XRd to run on.
-  #cpuset: ""
-  # Override the number of CPUs used for control-plane threads from
-  # the pool of CPUs in the XRd cpuset.
-  #controlPlaneCpuCount: 2
+  # Specify the cpusets for XRd to use for the control plane and the data
```

Comment:
"control-plane" and "dataplane" is preferred (this is consistent with the user guide and should be consistent with the rest of this value file)